olca-app/src/org/openlca/app/editors/parameters/RenameParameterDialog.java

```diff
@@ -86,7 +87,7 @@ protected void createFormContent(IManagedForm form) {
 		UI.gridData(comp, true, false);
 		UI.gridLayout(comp, 2, 10, 0);
 
-		text = UI.labeledText(comp, tk, "New name");
+		text = UI.labeledText(comp, tk, NLS.bind(M.New, M.Name));
```

Comment:
why NLS.bind here when the label is just `New name`?

Reply:
I see it used some time (cf. [here](https://github.com/GreenDelta/olca-app/blob/ba57e9d35cfab9b4de29e4b0da5f55e3b6c1bd4b/olca-app/src/org/openlca/app/devtools/SaveScriptDialog.java#L89)), I thought it was a way to avoid increasing the number of words.

Could you tell me when you would use it?